deploy/gk-deploy

```diff
@@ -476,10 +476,14 @@ fi
 
 if [[ ${LOAD} -eq 0 ]]; then
   if [[ "${CLI}" == *oc\ * ]]; then
-    eval_output "${CLI} process -p HEKETI_ADMIN_KEY=${ADMIN_KEY} -p HEKETI_USER_KEY=${USER_KEY} deploy-heketi | ${CLI} create -f - 2>&1"
+    oc_version = $(${CLI} version | grep "oc")
+    if [[ "${oc_version}" == *3.4.* ]]; then
```

Comment:
One thought here: which versions do we support?
If we support origin (I think we do), oc reports v1.4.0 or v1.5.X, so this check would not apply.
We need to check for at least two flavors of oc version output  v1.X.Y and v3.X.Y.

It could be made more general like this:

```
VAL_SWITCH="-p"
oc_minor_version=$(oc version | grep oc | awk '{ print $2 }' | cut -d '.' -f2)
if [[ ${oc_minor_version} -lt 5 ]]; then
  VAL_SWITCH="-v"
fi
eval_output "${CLI} process ${VAL_SWITCH} HEKETI_ADMIN_KEY=${ADMIN_KEY} ${VAL_SWITCH} HEKETI_USER_KEY=${USER_KEY} deploy-heketi | ${CLI} create -f - 2>&1"
```